### ISSUE TITLE
Ability to remove toolbar and replace icons with text labels

### ISSUE THREAD

**Issue by accessibility-bug-reports** [keepassxreboot/keepassxc#2343]
Visual agnosia is a vision impairment that makes one unable to recognize things. Like icons. Also, a variety of conditions can cause difficulty discerning what tiny icons represent. This is a feature request for either the ability to remove the icons-only toolbar, to have the icons replaced with text labels, or preferably both.

## Expected Behavior
Ideally, one would go into the settings and either check or uncheck "Show toolbar" or check "Icons only", "Icons and text", or "Text only". This would give more freedom in how people prefer to have their buttons, for whatever reason, or remove them entirely, if, for example, the system font size was large enough to make a toolbar impractical.

## Current Behavior
At present, there is no apparent way to edit the toolbar, remove the toolbar, or change how the buttons on the toolbar are displayed. There is nothing in the settings menu, nor can you right click on the toolbar or a button to edit, which is more or less the two main ways of accomplishing this. (I can't find any settings about this anywhere, either.)

## Possible Solution
I wish I knew enough to offer a potential way to fix this. 

## Context
I find the toolbar distracting. It also makes the program look more cluttered. I'd love if it could be removed entirely, since I'm used to using the menus. Though some text buttons would be nice, fewer mouse clicks that way. But if you did that, you'd need to make sure that what's included in the toolbar can be edited, because otherwise it wouldn't fit with the font size I use. 

## Debug Info

KeePassXC - 2.3.4
Revision: 6fe821c

Libraries:
- Qt 5.11.1
- libgcrypt 1.8.3

Operating system: Manjaro Linux
CPU architecture: x86_64
Kernel: linux 4.17.19-1-MANJARO

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by boppybibbles** [comment]
@accessibility-bug-reports, as for version 2.4.0 there is a checkbox in settings to hide that toolbar.

Also I've tried experimenting with UI and looks like it is possible to get what you want with icons and labels. See screenshots below. I'll see if I will be able to make a proper PR for this.
 
![image](https://user-images.githubusercontent.com/43829320/46438281-2a980e80-c766-11e8-935d-99e8014b03ba.png)

![image](https://user-images.githubusercontent.com/43829320/46438946-ef96da80-c767-11e8-901f-3d67d569f7d8.png)

**Comment by accessibility-bug-reports** [comment]
@boppybibbles Sorry I wasn't clear, I meant this toolbar: 

![screenshot_20181003_213013](https://user-images.githubusercontent.com/43528857/46448580-14df1500-c777-11e8-86e1-3dc557b9a0d7.png)

I almost never see that screen, since I have it set to remember the database. I didn't remember that there was a toolbar there. 

I like the second image would be ideal, but both would work. I really appreciate it.

**Comment by boppybibbles** [comment]
The toolbar is the same entity no matter which screen is visible at the moment.

**Comment by EasyReader2** [comment]
Hello droidmonkey,
I use Ubuntu MATE and Manjaro MATE. These desktops use GTK+. I just love big icons / buttons in the toolbar. With GTK+ applications, I have no problem. With qt5 applications, there are the following tools to help configure the appearance on GTK+: qt5ct and Kvantum Manager. I managed to use qt5ct to set the font siez to 14, which helped, but then the original keepassxc icons were replaced by my theme icons. I renamed the respective qt5 config files in ~/.config and font size remained 14 (good!) and the orignal icons were back again (great!). Kvantum Manager did not help much in order to configure icon sizes in keepassxc. The icons in the toolbar are too small form my taste, to see and to click. Another painfull example for small tool icons is notepadqq. If I remember correctly, Inkscape offers the possibility to scale up the icons in the toolbar. Could you give as such a possibilty in keepassxc too? Many thanks for your help.

**Comment by droidmonkey** [comment]
Our work on the new UI design in 2.6.0 will likely solve all your issues here!

**Comment by EasyReader2** [comment]
Thanks for the quick reply. So I am really looking forward to the new release :-)
Many thanks to the entire team for a this great native, truely platform-independent tool.  I really appreciate your work.
